### PR TITLE
fix: org delete commands to sf style

### PR DESCRIPTION
**Pull request by CristiCanizales**
### What does this PR do?
- Updates org delete commands and flags to SF style in order to get rid of cli warnings

### What issues does this PR fix or reference?
@W-14700710@

### Functionality Before
- org delete commands in sfdx style

### Functionality After
- org delete commands in sf style

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/commands/orgDelete.ts [hunk 1]
```diff
@@ -30,15 +30,15 @@ export class ForceOrgDeleteExecutor extends SfdxCommandletExecutor<{}> {
 
   public build(data: { choice?: string; username?: string }): Command {
     const builder = new SfdxCommandBuilder()
-      .withDescription(nls.localize('force_org_delete_default_text'))
-      .withArg('force:org:delete')
-      .withArg('--noprompt')
-      .withLogName('force_org_delete_default');
+      .withDescription(nls.localize('org_delete_default_text'))
+      .withArg('org:scratch')
```
Comment (peternhale) [inline packages/salesforcedx-vscode-core/src/commands/orgDelete.ts:34]:
``` suggestion
      .withArg('org:delete:scratch')
```

**Review by peternhale — CHANGES_REQUESTED** [review]

File: packages/salesforcedx-vscode-core/src/commands/orgDelete.ts (hunk 1 above)
Comment (CristiCanizales) [reply packages/salesforcedx-vscode-core/src/commands/orgDelete.ts:34]: I wonder how tests were passing with this typo... 🤔

Comment (peternhale) [reply packages/salesforcedx-vscode-core/src/commands/orgDelete.ts:34]: Integration tests disabled.

**Review by peternhale — APPROVED** [review]

**Review by peternhale — APPROVED** [review]

**Review by daphne-sfdc — APPROVED** [review]
Approved! 🎉

Commands Tested:

1. SFDX: Delete Default Org ✅
2. SFDX: Delete Org ✅

**Pull request merged**